subsys/bluetooth/services/Kconfig.ots

```diff
@@ -0,0 +1,52 @@
+#
+# Copyright (c) 2020 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+#
+
+config BT_GATT_OTS
+	bool "Object Transfer Service"
+	select BT_NRF_SERVICES
+	select BT_L2CAP_DYNAMIC_CHANNEL
+	select BT_GATT_DYNAMIC_DB
+	select BT_SMP
+	help
+	  Enable Object Transfer Service.
+
+if BT_GATT_OTS
+
+config BT_GATT_OTS_MAX_INST_CNT
+	int "Maximum number of available OTS instances"
+	default 1
+
+config BT_GATT_OTS_MAX_OBJ_CNT
+	int "Maximum number of objects that server can store"
+	default 5
+
+config BT_GATT_OTS_MAX_NAME_LEN
+	int "Maximum name length of an object"
+	default 40
```

Comment:
One advantage of keeping the object data in the application, rather in OTS, is that it is much easier for each appplication to optimize memory usage. E.g. if we have 2 applications/primary services that want to use OTS, but they differ vastly in how long names their objects have, then we use extra memory as we have to support the largest value for all applications/primary services. 

I don't think doing it with Kconfig is a bad in any way, but I'm just a bit uncertain whether or not it is the right way that we are storing the data.

Reply:
The more data is stored exclusively in the application, the more can be placed in flash rather than RAM as well

Reply:
I decided to remove this configuration because it is not that useful. Object name data will be provided by the application via pointer.